fusedrug/data/tokenizer/modulartokenizer/README.md

```diff
@@ -10,8 +10,12 @@ Creates a modular tokenizer that combines multiple initial tokenizers, adjusting
 * They all share the same special tokens (combined special tokens from all the source tokenizers),
 * Each tokenizer retains its regular tokens, however their IDs are remapped to a single space, with no overlaps.
 Note: If a token has the same meaning across all input types (e.g. special tokens, like SEP, EOS, sentinel tokens), it should be defined as special token in at least one of the initial (input) tokenizers.
+### save():
+Saves all mudular tokenizer information to a given path.
 ### save_jsons():
```

Comment:
Do we want to keep both save and save_jsons?
If you only recommend to use save() we can remove save_jsons from README.

Reply:
Removed from readme and commented in the examples